app/src/main/java/com/example/moodtrackerproject/routing/Route.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.moodtrackerproject.routing
+
+import android.content.Context
+import android.content.Intent
+import androidx.fragment.app.FragmentActivity
+import com.example.moodtrackerproject.MainActivity
+import com.example.moodtrackerproject.R
+import com.example.moodtrackerproject.ui.LoginFragment
+
+object Route {
+    fun goToLogin(fragmentActivity: FragmentActivity) {
+        val transaction = fragmentActivity.supportFragmentManager.beginTransaction()
+        transaction.replace(R.id.nav_host_fragment, LoginFragment())
+        transaction.commit()
+    }
+    fun goToMainActivity(context: Context) {
```

Comment:
`context ` should be Activity context, otherwise your app will crash asking to add Intent flag NEW_TASK.
smth like https://stackoverflow.com/questions/52687908/startactivity-in-java-class-crash